bolt-design-system/bolt
# Bolt start command on external projects

## Jira

http://vjira2:8080/browse/BDS-1086

## Summary

Give the possibility to run bolt start on an external project.

## Details

Update watched files on Browser Sync configuration by excluding `node_modules` from glob patterns.

## How to test

Run this code locally. Go to `example-integrations/static-html/.bolt` and run `yarn` command. When all dependencies are installed correctly run `npm start` command and check if this command run `bolt start` without errors. Update `index.html` located in `example-integrations/static-html/` to check if the watch is working correctly. 

_**TODO**: Consider to add to excluded files `vendor` dir (Drupal 8 dependencies)._
